README.md

```diff
@@ -90,6 +90,8 @@ Interface | RX | TX
 **Serial 2**   | GPIO 35   | GPIO 33
 **CAN**   | GPIO 5   | GPIO 4
 
+Further description can be found [here](https://github.com/shining-man/bsc_fw/wiki/BSC%E2%80%90Test-ohne-orig.-Hardware).
```

Comment:
Link hatte sich in der Zwischenzeit geändert -> https://github.com/shining-man/bsc_fw/wiki/40-BSC%E2%80%90Test-ohne-orig.-Hardware